webpack.config.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'node:path'
+import { Configuration } from 'webpack'
+
+const config: Configuration = {
+  context: path.resolve(__dirname),
+  devtool: 'source-map',
+  entry: './src/client/index.ts',
+  mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.(m?js|ts)$/,
+        exclude: /node_modules/,
+        use: ['babel-loader'],
+      },
+    ],
+  },
+  output: {
+    library: {
+      name: 'MODUK',
+      type: 'umd',
+    },
+    filename: 'moduk-frontend.umd.js',
+    path: path.resolve(__dirname, './dist/client'),
+  },
+  resolve: {
+    extensions: ['.ts', '.mjs', '..'],
```

Comment:
Should be `...` [according to the docs](https://webpack.js.org/configuration/resolve/#resolveextensions):

```suggestion
    extensions: ['.ts', '.mjs', '...'],
```

Reply:
🤦